runtime/src/cost_tracker.rs

```diff
@@ -106,8 +143,9 @@ impl CostTracker {
                 return;
             }
 
-            let adjustment =
-                i128::from(actual_execution_units) - i128::from(estimated_execution_units);
+            let (res, _overflowed) =
+                actual_execution_units.overflowing_sub(estimated_execution_units);
+            let adjustment = res as i64;
```

Comment:
we need to respond to both overflow and `res` too large for `i64` here as well

Reply:
I can't find any way better than what @ckamm suggested above if have to take care of `res` is too big for `i64`. But sounds like the consensus is `saturating` is not the ideal solution. I hesitant to copy `saturating_add_unsigned` implementation into our code space for an undesired solution. 

ig if we are going to compare `a` and `b` before math, we may as well check `if abs(a-b)> i64::MAX`. Then the real question is if it's true, 1. what to do with the presumably bad adjustment, throw it away? 2. what to do with current block, stop adding more txs into it to be safe, or reset its cost to zero therefore risking making a heavy block (and  long replay time)?

Reply:
Ya, OK sounds good. changes in [commit](89397ea8d43a73a637b31ad58662a88d84b2f53a)